worlds/dark_souls_3/Locations.py

```diff
@@ -688,6 +688,273 @@ def get_name_to_id() -> dict:
+        "US: Soul of the Rotted Greatwood",
+        "RS: Soul of a Crystal Sage",
+        "FK: Soul of the Blood of the Wolf",
+        "FK: Cinders of a Lord - Abyss Watcher",
+        "CD: Small Doll",
+        "CD: Soul of the Deacons of the Deep",
+        "CD: Archdeacon White Crown",
+        "CD: Archdeacon Holy Garb",
+        "CD: Archdeacon Skirt",
+        "CC: Soul of High Lord Wolnir",
+        "SL: Soul of the Old Demon King",
+        "IBV: Soul of Pontiff Sulyvahn",
+        "PC: Soul of Yhorm the Giant",
+        "PC: Cinders of a Lord - Yhorm the Giant",
+        "AL: Soul of Aldrich",
+        "AL: Cinders of a Lord - Aldrich",
+        "HWL: Soul of the Dancer",
+        "LC: Soul of Dragonslayer Armour",
+        "CKG: Soul of Consumed Oceiros",
+        "UG: Soul of Champion Gundyr",
+        "UG: Coiled Sword Fragment",
+        "GA: Soul of the Twin Princes",
+        "GA: Cinders of a Lord - Lothric Prince",
+        "AP: Soul of the Nameless King",
+        "PW: Soul of Sister Friede",
+        "PW: Valorheart",
+        "PW: Champion's Bones",
+        "DH: Soul of the Demon Prince",
+        "DH: Small Envoy Banner",
+        "RC: Soul of Darkeater Midir",
+
+        # Not currently randomized
+        # "FS: Coiled Sword",
+        # "AP: Dragon Head Stone",
+        # "RC: Fillianore's Spear Ornament",
+        # "RC: Spears of the Church",
+    ]),
+
+    "Miniboss Rewards": frozenset([
+        "FS: Uchigatana", # NPC drop
+        "FK: Soul of a Stray Demon",
+        "US: Irithyll Straight Sword", # Irithyll Outrider drop
+        "CC: Soul of a Demon",
+        "SL: Lightning Stake", # Sand Worm drop
+        "SL: Fume Ultra Greatsword", # Knight Slayer Tsorig drop
+        "CD: Aldrich's Sapphire", # Deep Accursed drop
+        "CD: Spider Shield", # NPC drop
+        "CD: Saint Bident", # Guarded by giant
+        "IBV: Pontiff's Right Eye", # Sulyvahn's Beast drop
+        "IBV: Ring of Favor", # Sulyvahn's Beast Duo drop
+        "IBV: Aldrich Faithful", # Sulyvahn's Beast Duo reward
+        "AL: Aldrich's Ruby", # Deep Accursed drop
+        "PC: Logan's Scroll", # NPC drop
+        "LC: Irithyll Rapier", # Boreal Outrider drop
+        "GA: Crystal Scroll", # Crystal Sage drop
+        "GA: Outrider Knight Helm",
+        "GA: Outrider Knight Armor",
+        "GA: Outrider Knight Gauntlets",
+        "GA: Outrider Knight Leggings",
+        "GA: Golden Wing Crest Shield", # NPC drop
+        "GA: Onikiri and Ubadachi", # NPC drop
+        "GA: Sage's Crystal Staff", # NPC drop
+        "AP: Dragon Tooth", # NPC drop
+        "DH: Flame Fan", # Desert Pyromancer Zoey drop
+        "RC: Iron Dragonslayer Helm",
+        "RC: Iron Dragonslayer Armor",
+        "RC: Iron Dragonslayer Gauntlets",
+        "RC: Iron Dragonslayer Leggings",
+        "RC: Crucifix of the Mad King", # Shira drop
+    ]),
+
+    "Key Locations": frozenset([
+        "HWL: Cell Key",
+        "ID: Jailer's Key King",
```

Comment:
```suggestion
        "ID: Jailer's Key Ring",
```

Fixed to match the actual location name